Cargo.toml

```diff
@@ -14,6 +14,10 @@ license = "Apache-2.0"
 [badges]
 travis-ci = { repository = "denisidoro/navi", branch = "master" }
 
+[features]
+default = []
+use_skim = []
```

Comment:
Where is the skim dependency?

Reply:
It is not dependent on skim during compilation. It just requires the binary. Works in the same way as fzf.